### PR TITLE
When getting local value, we don't need to copy every value

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: build/walrus.cmake [hunk 1]
```diff
@@ -68,6 +68,7 @@ ELSEIF (${WALRUS_MODE} STREQUAL "release")
 ENDIF()
 
 SET (WABT_DEFINITIONS ${WALRUS_DEFINITIONS})
+SET (WITH_EXCEPTIONS TRUE)
```
Comment (clover2123) [inline build/walrus.cmake:71]: Where is this flag used for?

Comment (ksh8281) [reply build/walrus.cmake:71]: I want to throw exception in wabt code, but wabt don't enable exception feature default.

Comment (clover2123) [reply build/walrus.cmake:71]: I see :)

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**